docs/ref/utils.txt

```diff
@@ -556,6 +556,14 @@ https://web.archive.org/web/20110718035220/http://diveintomark.org/archives/2004
         z = person.friends         # does not call
         x is z                     # is True
 
+.. class:: classproperty(method=None)
+
+    .. versionadded:: 3.1
+
+    The ``@classproperty`` decorator converts the result of a method with a
+    single ``cls`` argument into a property that can be accessed directly from
+    the class.
+
```

Comment:
**Maybe** it would be worth mentioning `@classmethod` here? 
("Maybe...")

```suggestion
    Similar to :py:func:`classmethod`, the ``@classproperty`` decorator
    converts the result of a method with a single ``cls`` argument into a
    property that can be accessed directly from the class.
```

(Didn't test the rst-link.)

Reply:
Thanks, added.

Reply:
Thank you both for promptly bringing this to closure 😄